index.html

```diff
@@ -1,14 +1,244 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Homework 6</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="Styles/style.css">
```

Comment:
add a link for reset.css or remove spaces in style.css

Reply:
removed